d2l-dom-expand-collapse.html

```diff
@@ -31,7 +31,7 @@
 					node.style.overflow = 'hidden';
 					node.style.transition = '';
 					node.style.height = '0px';
-					node.style.display = node.getAttribute('data-d2l-ec-display');
+					node.style.display = node.getAttribute('data-d2l-ec-display') ? !node.getAttribute('data-d2l-ec-display') : '';
```

Comment:
I think I had a copy/paste error in my original suggestion.  I think you want:
```
node.style.display = node.getAttribute('data-d2l-ec-display') ? node.getAttribute('data-d2l-ec-display') : '';
```
i.e. remove the `!` from the 2nd argument.

Reply:
Oh yes right, my mistake too! If `node.getAttribute('data-d2l-ec-display')` is not null we want it!